package.json

```diff
@@ -49,7 +49,7 @@
     "e2e:api": "cross-env USE_API_SETTINGS=true start-server-and-test e2e:serve http://localhost:3000 cy:run:api",
     "cy:open": "cross-env TZ=UTC cypress open",
     "cy:run": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_mock_data/*.cy.ts\"",
-    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/*.cy.ts\""
+    "cy:run:api": "cross-env TZ=UTC cypress run --spec \"./cypress/e2e/with_api/\""
```

Comment:
Why did this need changing? I thought it only specified the files to run?

Reply:
Ah I see, can you use `./cypress/e2e/with_api/**/*.cy.ts` instead so it doesn't attempt to look at the functions files, it looks like it finds the subdirectories and `app.cy.ts` as well when I tested it.